src/cli/main.c

```diff
@@ -112,6 +113,10 @@ parser(int key, char *arg, struct argp_state *state)
         case 'l':
                 ctx->ldcache = arg;
                 break;
+        case 0x80:
+                if (str_join(&err, &ctx->init_flags, "no-create-imex-channels", " ") < 0)
+                        goto fatal;
+                break;
```

Comment:
Why is this a top-level option, rather than specific to the configure / info calls?

Reply:
I think I answered my own question -- because it needs to be used during `nvc_init()` as part of "loading the kmods" (which itself is a top-level option).

Reply:
Yes, this option is "parallel" to `load-kmods`.

Reply:
OK; let's keep it where it is then.